less/index.less

```
@@ -48,12 +48,280 @@ table {
 	border-spacing: 0;
 }
 
+//----VARIABLES-----
+@resume-blue: #E1EAEC;
+@resume-font: arial;
+//----VARIABLES-----
+
+// ---------- MIXINS -----------
+
+.centerX() {
+	justify-content: center;
+}
+
+.centerY() {
+	align-items: center;
+}
+
+.fullwidth() {
+	width: 100%;
+}
+
+// ---------- MIXINS -----------
+
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	font-family: @resume-font;
 }
 
+.container {
+
+	font-size: 1.6rem;
+	max-width: 1000px;
+	margin: 0 auto;
+	;
+
+	h1 {
+		font-size: 5rem;
+	}
+	
+	p {
+		font-size: 1.6rem;
+	}
+	
+	nav {
+		display: flex;
+		width: 35%;
+		justify-content: space-between;
+
+		a {
+			text-decoration: none;
+			color: black;
+		}
+	}
+
+	header {
+		display: flex;
+		flex-direction: row-reverse;
+		.fullwidth;
+		margin: 20px auto;
+
+		@media(max-width: 500px) {
+
+			display: flex;
+			align-items: center;
+			.centerX;
+
+			nav {
+				width: 100%;
+				display: flex;
+				flex-direction: column;
+				flex-wrap: wrap;
+	
+
+				a {
+					width:100%;
+					color: black;
+					border: 1px solid grey;
+					padding: 10px;
+					font-size: 2rem;
+					margin: 2px auto;
+					text-align: center;
+				}
+
+				a:hover {
+					background-color: grey;
+				}
+			}
+		}
+
+	}
+
```

```suggestion
	header {
		display: flex;
		flex-direction: row-reverse;
		.fullwidth;
		margin: 20px auto;

		@media(max-width: 500px) {

			display: flex;
			align-items: center;
			.centerX;

			nav {
				width: 100%;
				display: flex;
				flex-direction: column;
				flex-wrap: wrap;


				a {
					width:100%;
					color: black;
					border: 1px solid grey;
					padding: 10px;
					font-size: 2rem;
					margin: 2px auto;
					text-align: center;
				}

				a:hover {
					background-color: grey;
				}
			}
		}

	}

```
Really nice nesting here.